stdlib/public/stubs/GlobalObjects.cpp

```diff
@@ -42,6 +42,7 @@ swift::_SwiftEmptyArrayStorage swift::_swiftEmptyArrayStorage = {
   // HeapObject header;
   {
     &swift::CLASS_METADATA_SYM(s19__EmptyArrayStorage), // isa pointer
+    InlineRefCounts::Immortal
```

Comment:
Will this cause these objects not to get dirtied at all? That would be 👌

Reply:
Unfortunately iirc the isa fixup will still dirty the page